src/jit/lsra.h

```diff
@@ -1131,7 +1131,7 @@ class LinearScan : public LinearScanInterface
     int compareBlocksForSequencing(BasicBlock* block1, BasicBlock* block2, bool useBlockWeights);
     BasicBlockList* blockSequenceWorkList;
     bool            blockSequencingDone;
-    void addToBlockSequenceWorkList(BlockSet sequencedBlockSet, BasicBlock* block);
+    void addToBlockSequenceWorkList(BlockSet sequencedBlockSet, BasicBlock* block, BlockSet predSet);
     void removeFromBlockSequenceWorkList(BasicBlockList* listNode, BasicBlockList* prevNode);
```

Comment:
Although I guess you don't want to pass by value, do you? Should it be `BlockSet&`?

---
In reply to: [113554764](https://github.com/dotnet/coreclr/pull/11233#discussion_r113554764) [](ancestors = 113554764)